packages/storybook/stories/controlsUtils.ts

```diff
@@ -133,8 +133,8 @@ export const getControlledTheme = (choice: string): PartialTheme => {
   return DefaultTheme;
 };
 
-const VIDEO_GALLERY_LAYOUTS = ['default', 'floatingLocalVideo'] as const;
-const OVERFLOW_GALLERY_LAYOUTS = ['HorizontalBottom', 'VerticalRight'] as const;
+const VIDEO_GALLERY_LAYOUTS = ['default', 'floatingLocalVideo', 'speaker'] as const;
```

Comment:
focusedContent also here?

Reply:
Just adding to the controls as needed, will come when the layout is made 😊

Reply:
Oh, should this layout not be here then
![image](https://github.com/Azure/communication-ui-library/assets/2684369/2b0fc450-3292-458e-8942-0d20e81209d6)

Reply:
good point, will remove until the component is made!